Soccar [Game-Server]/src/nl/soccar/gameserver/controller/socnet/message/JoinSessionMessage.java

```diff
@@ -43,22 +64,47 @@ public JoinSessionMessage(Status status, String roomName, int capacity, GameSett
         this.settings = settings;
     }
 
+    /**
+     * Gets the Room name.
+     * 
+     * @return String The room name.
+     */
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public enum. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)